modules/031-linstor/docs/FAQ.md

```diff
@@ -62,13 +62,13 @@ You can also add pools with some volumes have already been created. LINSTOR will
 
 ## How to configure Prometheus to use LINSTOR for storing data?
 
-Configure storage-pools and StorageClass, as shown in [Usage](usage.html)
+Configure storage-pools and StorageClasses, as shown in [Usage](usage.html)
 
 Add to the Deckhouse configuration (configMap `d8-system/deckhouse`):
 ```yaml
 prometheus: |
-  longtermStorageClass: linstor-r2
-  storageClass: linstor-r2
+  longtermStorageClass: linstor-data-r2
+  storageClass: linstor-data-r2
```

Comment:
Since the storageclasses will be generated automatically at now, they always have the following name template `linstor-<some_name>-r<1|2|3>`.

`data` is the most typical name for storagepool, do you think we should change it?